src/modules/blockchainApplication/connection/components/ConnectionSummary/index.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import { withRouter } from 'react-router';
+import ValueAndLabel from 'src/modules/transaction/components/TransactionDetails/valueAndLabel';
+import { PrimaryButton, SecondaryButton } from 'src/theme/buttons';
+import { EVENTS } from '@libs/wcm/constants/lifeCycle';
+import { addSearchParamsToUrl } from 'src/utils/searchParams';
+import ConnectionContext from '@libs/wcm/context/connectionContext';
+import useSession from '@libs/wcm/hooks/useSession';
+import { useAccounts } from '@account/hooks';
+import BlockchainAppDetailsWrapper from '../../../explore/components/BlockchainAppDetailsWrapper';
+import AccountsSelector from './AccountsSelector';
+import styles from './connectionSummary.css';
+
+// eslint-disable-next-line max-statements
+const ConnectionSummary = ({ history }) => {
+  const [addresses, setAddresses] = useState([]);
+  const { t } = useTranslation();
+  const { accounts } = useAccounts();
```

Comment:
accounts dosnt have use case here, please move it to AccountsSelector

Reply:
Doen